src/main/webapp/creatives/maps/simid_maps.js

```diff
@@ -1,8 +1,47 @@
 /**
  * A sample SIMID ad that shows a map of nearby locations
  */
+const AdParamKeys = {
+  BUTTON_LABEL: 'buttonLabel',
+  SEARCH_QUERY: 'searchQuery',
+  MARKER: 'marker',
+};
+
+const FIND_NEAREST = "Find Nearest ";
```

Comment:
FIND_NEAREST_TEMPLATE_TEXT or something else more descriptive

Reply:
got it